main.go

```diff
@@ -1,12 +1,22 @@
 package main
 
 import (
+	"fmt"
 	"os"
 
 	"github.com/gittuf/gittuf/internal/cmd/root"
 )
 
 func main() {
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Printf("unexpected error: %s\n\n", fmt.Sprint(r))
+			fmt.Printf("Please consider opening an issue on https:/github.com/gittuf/gittuf/issues tagged 'bug' with the steps to reproduce this state. Thanks!\n")
```

Comment:
If you want to make this even easier, you can create a bug template to auto-label this: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

Reply:
Added a bug report template.